modules/automod/automod.ts

```diff
@@ -14,6 +14,25 @@ import warn from "../punishments/warn.js";
 import tryCensor, { badWordRegexps, badWordsAllowed } from "./misc.js";
 import { stripMarkdown } from "../../util/markdown.js";
 import { joinWithAnd } from "../../util/text.js";
+import { createWorker } from "tesseract.js";
+const worker = await createWorker("eng");
+async function getMessageImageText(message: Message): Promise<string[]> {
+	const imageUrls: string[] = message.attachments
+		.filter((attachment) => attachment?.contentType?.match(/^image\/(bmp|jpeg|png|bpm|webp)$/i))
+		.map((attachment) => attachment.url);
+
+	const imageTextPromises = imageUrls
+		.map(async (url) => {
```

Comment:
```suggestion
		.map(async ({url}) => {
```
fix variable names ofc